modules/e2ee/OlmAdapter.js

```diff
@@ -207,6 +226,27 @@ export class OlmAdapter extends Listenable {
         }
     }
 
+    /**
+     * Sends sacMac if channel verification waas successful.
+     *
+     */
+    markParticipantVerified(isVerified, participant) {
+        if (isVerified) {
```

Comment:
Else, we need to destroy everything we have regarding this SAS, don't we? Also we need to support trying again I think.